index.js

```diff
@@ -73,6 +73,6 @@ function fastifyGracefulShutdown(fastify, opts, next) {
 }
 
 module.exports = fp(fastifyGracefulShutdown, {
-  fastify: '^1.0.0 || ^2.0.0',
+  fastify: '^1.0.0 || ^2.0.0 || ^3.0.0',
```

Comment:
I think you misunderstood my review. It's fine to support only `^3.0.0` and add `fastify-plugin@2.0.0` which adds support for fastify v3.

Reply:
Fixed it